pkg/tuned/cmd/render/cmd.go

```diff
@@ -0,0 +1,85 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package render
+
+import (
+	"flag"
+	"fmt"
+
+	"github.com/openshift/cluster-node-tuning-operator/pkg/performanceprofile/controller/performanceprofile/components"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+
+	"k8s.io/klog"
+)
+
+type renderOpts struct {
+	assetsInDir  string
+	assetsOutDir string
+}
+
+func NewRenderBootCmdMCCommand() *cobra.Command {
+	renderOpts := renderOpts{}
+
+	cmd := &cobra.Command{
+		Use:   "render-bootcmd-mc",
+		Short: "Render MC with kernel args",
+		Run: func(cmd *cobra.Command, args []string) {
+			if err := renderOpts.Validate(); err != nil {
+				klog.Fatal(err)
+			}
+
+			if err := renderOpts.Run(); err != nil {
+				klog.Fatal(err)
+			}
+		},
+	}
+
+	addKlogFlags(cmd)
+	renderOpts.AddFlags(cmd.Flags())
+	return cmd
+}
+
+func (r *renderOpts) AddFlags(fs *pflag.FlagSet) {
+	fs.StringVar(&r.assetsInDir, "asset-input-dir", components.AssetsDir, "Input path for the assets directory. (Can be a comma separated list of directories.)")
```

Comment:
Comma is a valid (although seldom) used character for paths. What about allowing multiple occurences of asset-input-dir with one path each?

Reply:
Just following the same "interface" as the PerformanceProfile render command 
https://github.com/openshift/cluster-node-tuning-operator/blob/cd53f65cdbc8e64210a632f7739410fd7697ef7f/pkg/performanceprofile/cmd/render/cmd.go#L63

Reply:
The interface there was arbitrary, there is no standard way of writing the render commands. I would feel better if we used the typical shell pattern of repeated arguments (at both places really...).

Reply:
Changed to allow multiple instances of the `asset-input-dir` flag.